src/main/java/CommandChecker.java

```diff
@@ -0,0 +1,63 @@
+import java.util.ArrayList;
+
+public class CommandChecker {
+
+    public void checkCommand(String command, ArrayList<Task> list) throws DukeException {
+        if (command.length() < 4) {
+            throw new DukeException("☹ OOPS!!! I'm sorry, your command is too short \uD83D\uDE09 \n");
+        } else if (command.contains("todo") || command.contains("event") || command.contains("deadline") || command.contains("delete") || command.contains("done")) {
```

Comment:
Hello! I think your code is generally well written, just take note that this line appears a bit long so its good to split it into 2!